### ISSUE TITLE
Tracking numbers not supplied to customer

### ISSUE THREAD

**Issue by jessepearson** [Automattic/woocommerce-services#1865]
**Affected Tickets**
2621038-zen
18307011-hc

**Describe the bug**
Customer has purchased shipping labels, however tracking numbers are not available. RC for the customer showing the label purchases can be found in the ticket. I have not linked here due to this is a public repo. 

There was a previous instance where this was a one off:
https://a8c.slack.com/archives/C074153QS/p1548423136690100

**To Reproduce**
1. Unsure of how to reproduce

**Expected behavior**
I would expect that tracking numbers would be provided for purchased labels.

**Comment by aleftick** [comment]
I had quick look at a few labels and they are all letter mail (First-Class Mail). Letter mail aren't tracked by USPS; it's mailing an envelope with a stamp. If the merchant wants tracking, he needs to upgrade to a shipping service that includes it: Priority, Priority Express of First-Class Package).

**Comment by jessepearson** [comment]
Okay, that makes sense. Maybe it would help customers and support if this was somehow shown to the customer and/or support.

**Comment by treibalen** [comment]
Hi there @aleftick,

> I had quick look at a few labels and they are all letter mail (First-Class Mail). Letter mail aren't tracked by USPS; it's mailing an envelope with a stamp. 

I had another report that USPS First Class Mail says that it provides tracking while it doesn't eventually: 2662164-zen

Indeed, I do see that tracking is mentioned for USPS First Class Mail with envelope package on https://a8c-wcs-support.mystagingwebsite.com/wp-admin/post.php?post=3314&action=edit#

![https://d.pr/i/0p792S](https://d.pr/i/0p792S+) 
Full Size: https://d.pr/i/0p792S

Isn't it a bug then if USPS First Class Mail letters don't provide tracking?

**Comment by aleftick** [comment]
Yes @treibalen it looks like a bug to me and I was able to reproduce it.

Hopefully, the EasyPost API can help differentiate First Class Mail vs First Class Package so that we can hard code the "Includes USPS tracking" message. 
However, I suspect they don't. It's the same USPS service and not always differentiated. So, if there is no way to tell from the EasyPost API response, I can suggest two ideas:
1. Could we use the package type? When merchants create a package, they can say if it's an envelope (therefore not display it has USPS tracking for FCM) or a box (therefore display it has USPS tracking for FCM). This would rely on the user declaring the right package type.
2. Can we look at the package dimensions; anything thicker than 3/4 inch or 15 inches in any dimension, would be considered a box by USPS and would have USPS tracking for FCM? [Here are the official USPS size restrictions](https://pe.usps.com/text/dmm100/mailing-domestic.htm).

Curiously, I got different rates when I compared custom packages with the same L:6.25, W:3, H:0.1, and 
 Weight of 0.01 (of the package empty and total) depending on if it was a `Box` vs `Envelope`. Do we use the package type for getting rates?
